packages/plugin-dev/src/node/hosted-instance-manager.ts

```diff
@@ -131,7 +135,8 @@ export abstract class AbstractHostedInstanceManager implements HostedInstanceMan
         let processOptions: cp.SpawnOptions;
         if (pluginUri.scheme === 'file') {
             processOptions = { ...PROCESS_OPTIONS };
-            processOptions.env.HOSTED_PLUGIN = pluginUri.path.toString();
+            // get filesystem path that work cross operating systems
+            processOptions.env.HOSTED_PLUGIN = await this.fileSystem.getFsPath(pluginUri.toString());
```

Comment:
just use `FileUri` you are on the backend it is safe to use node apis,  `FileSystem` should be used mostly on the frontend